libraries/libfc/include/fc/crypto/bls_private_key.hpp

```diff
@@ -28,6 +28,7 @@ namespace fc::crypto::blslib {
 
          bls_public_key     get_public_key() const;
          bls_signature      sign( const std::vector<uint8_t>& message ) const;
+         std::string        get_pop_str() const;

```

Comment:
Instead of returning the base64 encoded string of the `pop_prove` g2, I think this should be called `pop_proof()` and it should return a `bls_signature`.

Then in leap-util you can convert it into the `SIG_BLS_` prefixed base64 encoded string with a checksum.